winui/Ribbon/RibbonQAT.md

```diff
@@ -0,0 +1,233 @@
+                <ribbon:QuickAccessToolBar />
+            </ribbon:SfRibbon.QuickAccessToolBar>
+</ribbon:SfRibbon>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Adding items in Quick Access Toolbar
+
+Ribbon items can be added in the Quick Access Toolbar (QAT) by following below ways:
+
+ * Adding items through ContextMenu.
+ * Adding items through Code.
+ * Adding items using QAT MenuItems.
+
+ N> Currently, Ribbon Gallery is not supported as a QAT add-in.
+
+### Add items through ContextMenu
+
+To add an item to the Quick Access Toolbar (QAT), right-click the required ribbon item and select `Add to Quick Access Toolbar`. The respective item will then be added to the QAT. 
+
+![Add items through context menu in QAT](Ribbon-QAT-images/add-items-in-qat.png)
+
+
+### Adding items through Code
+
+To add the ribbon items in Ribbon Quick Access Toolbar through code behind, you need to populate the QuickAccessToolbar `Items` collection. The below code shows how to add the ribbon items in `Quick Access Toolbar` of Ribbon.
+
+{% tabs %}
+{% highlight xaml %}
+
+ <ribbon:SfRibbon x:Name="ribbon" >
+            <ribbon:SfRibbon.QuickAccessToolBar>
+                <ribbon:QuickAccessToolBar Position="AboveRibbon">
+                    <ribbon:RibbonButton x:Name="SaveButton"
+                                         Content="Save"
+                                         Icon="Save"
+                                         Command="{Binding ButtonCommand}"
+                                         CommandParameter="Save"/>
+                    <ribbon:RibbonButton x:Name="undoButton"
+                                         Content="Undo"
+                                         Icon="Undo"
+                                         Command="{Binding ButtonCommand}"
+                                         CommandParameter="Undo"/>
+                    <ribbon:RibbonButton x:Name="printButton"
+                                         Content="Print"
+                                         Icon="Print"
+                                         Command="{Binding ButtonCommand}"
+                                         CommandParameter="Print"/>                       
+                </ribbon:QuickAccessToolBar>
+            </ribbon:SfRibbon.QuickAccessToolBar>
+</ribbon:SfRibbon>
+
+{% endhighlight %}
+{% endtabs %}
+
+![Default QAT item in Ribbon control](Ribbon-QAT-images/default-qat.png)
+
+### Adding items using QAT MenuItems
+
+The Ribbon also supports adding items to the QAT Menu items. To add items to the drop-down menu of the Quick Access Toolbar, use the `MenuItems` property of the Quick Access Toolbar. Items can be added to the QAT by making the selection.
+
+In 'MenuItems', define the ribbon items that are either already contain in the Ribbon Tabs or not in the Ribbon Tabs. If the item is not in the Ribbon Tab, specify all the required properties. Otherwise, providing the 'Content' is sufficient.

```

Comment:
@Kayalvizhi-Sivashanmugam, compare WPF QAT document for this and change[SynchronizeItem]